README.md

```diff
@@ -12,32 +15,59 @@ Thoth is deployed to the following [environments](docs/environments.md).
 
 ## Separation of Application Components and Application Configuration
 
-Each component of the Thoth Application is separated into an ArgoCD Application, there is a core Application
-that contains global configs and secrets.
+Each component of the Thoth Application is separated into an ArgoCD Application.
+The core Application that contains global configs and secrets along with the setup of the namespace based requirements.
```

Comment:
The core Application contains global configs and secrets along with the setup of the namespace based requirements.